queries/capnp/highlights.scm

```diff
@@ -82,11 +88,14 @@
   "extends"
   "namespace"
   "using"
+  (void)
```

Comment:
`@type.builtin` or `@keyword`? I don't know `void` in the context of capnp

Reply:
it's actually @constant.builtin, just fixed it in latest commit!
Void is the type, void the value :)